src/actions/enter.ts

```diff
@@ -124,9 +126,15 @@ export class EnterAction {
     ) {
       return !workspace.isReadOnly();
     }
-    if (curNode instanceof BlockSvg) return true;
     // Returning true is sometimes incorrect for icons, but there's no API to check.
-    if (curNode instanceof icons.Icon) return true;
+    if (
+      curNode instanceof BlockSvg ||
+      curNode instanceof icons.Icon ||
+      curNode instanceof comments.CommentBarButton ||
+      curNode instanceof comments.RenderedWorkspaceComment
+    ) {
+      return true;
+    }
     return false;
```

Comment:
```suggestion
    return curNode instanceof BlockSvg ||
      curNode instanceof icons.Icon ||
      curNode instanceof comments.CommentBarButton ||
      curNode instanceof comments.RenderedWorkspaceComment;
```

Could we do this, instead?